src/core/Header/Header.tsx

```diff
@@ -0,0 +1,206 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import '@itwin/itwinui-css/css/header.css';
+import SvgChevronRight from '@itwin/itwinui-icons-react/cjs/icons/ChevronRight';
+import SvgHelpCircularHollow from '@itwin/itwinui-icons-react/cjs/icons/HelpCircularHollow';
+import SvgMoreVertical from '@itwin/itwinui-icons-react/cjs/icons/MoreVertical';
+import SvgNotification from '@itwin/itwinui-icons-react/cjs/icons/Notification';
+
+import cx from 'classnames';
+import React from 'react';
+import { IconButton } from '../Buttons';
+
+import { useTheme } from '../utils/hooks/useTheme';
+import { DropdownMenu } from '../DropdownMenu';
+
+export type HeaderTranslations = {
+  notifications: string;
+  help: string;
+  myAccount: string;
+  moreOptions: string;
+};
+
+export type HeaderProps = {
+  /**
+   * Icon to the application.
+   */
+  appIcon: React.ReactNode;
+  /**
+   * Name of the application.
+   */
+  appName?: React.ReactNode;
+  /**
+   * Handler on application click
+   */
+  onApplication?: () => void;
+  /**
+   * `HeaderButton` array, chevron will be added between each.
+   */
+  breadcrumbsButtons?: React.ReactNode[];
+  /**
+   * Content to display in the center section of the header, should handle iui-slim class if used.
+   */
+  centerContent?: React.ReactNode;
+  /**
+   * Expects `UserIcon`, will be given a drop down if `userMenuItems` is also provided;
+   */
+  userIcon?: React.ReactNode;
+  /**
+   * Handler on notification click, will show notification bell if provided.
+   */
+  onNotification?: () => void;
+  /**
+   * Items in the help dropdown menu.
+   * Pass a function that takes the `close` argument (to close the menu),
+   * and returns a list of `MenuItem` components.
+   */
+  helpMenuItems?: (close: () => void) => JSX.Element[];
+  /**
+   * Items in the user dropdown menu.
+   * Pass a function that takes the `close` argument (to close the menu),
+   * and returns a list of `MenuItem` components.
+   */
+  userMenuItems?: (close: () => void) => JSX.Element[];
+  /**
+   * Items in the more dropdown menu.
+   * Pass a function that takes the `close` argument (to close the menu),
+   * and returns a list of `MenuItem` components.
+   */
+  moreMenuItems?: (close: () => void) => JSX.Element[];
+  /**
+   * Reduce height of the header;
+   * @default false
+   */
+  slim?: boolean;
+  /**
+   * Provide localized strings.
+   */
+  translatedHeader?: HeaderTranslations;
```

Comment:
`translatedHeader` is not very intuitive, so I would call it `headerTranslations` or `translatedStrings` or anything plural

Reply:
Changed to translatedStrings, however, we only have one string now to translate, but I wasnt sure if I was to change it to a string instead of an object or not, string is ("More options")

Reply:
I would say to leave it as an object if we have some new strings in the future.